AutoDarkModeApp/Pages/PageSettings.xaml

```diff
@@ -113,7 +113,7 @@
                 Margin="0,30,0,0"
                 FontFamily="Segoe UI"
                 FontSize="20"
-                Text="Language"
+                Text="{x:Static p:Resources.SettingsPageLanguageSectionTitle}" 
```

Comment:
Same with this one, Language is omitted on purpose, for more info see here
https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/issues/284#issuecomment-938956434